README.md

```diff
@@ -1195,6 +1195,7 @@ Update Time, five active automations, webhooks.
   * [MDB GO](https://mdbgo.com/) - Free hosting for one project with two weeks Container TTL, 500 MB RAM per project, SFTP - 1G disk space.
   * [Patr Cloud](https://patr.cloud/) — An easy-to-use cloud platform, among its paid services it offers to host three static sites for free.
   * [Serv00.com](https://serv00.com/) — 3 GB of free web hosting with daily backups (7 days). Support: Crontab jobs, SSH access, repositories (GIT, SVN, and Mercurial), support: MySQL, PostgreSQL, MongoDB, PHP, Node.js, Python, Ruby, Java, Perl, TCL/TK, Lua, Erlang, Rust, Pascal, C, C++, D, R, and many more.
+  * [Zeabur](https://zeabur.com/) - This is Software as a Service, not self-hosted. It has a free tier, not just a free trial. Pricing information is clearly visible without signup or phone calls. The submission mentions what is free. The service has contact details of those running it and a privacy policy.
```

Comment:
the point here is that you should describe your service and what is free, see other entries.

That said there is already a PR for this one, its pricing isnt clear so I'll close this one.

See https://github.com/ripienaar/free-for-dev/pull/3220